.github/workflows/ios-release.yml

```diff
@@ -9,41 +21,41 @@ on:
   workflow_call:
     inputs:
       tag:
-        description: 'Tag/version (ex: 5.0.1)'
+        description: 'The existing tag (version) to be released (ex: 5.0.1).'
         required: true
         type: string
 
-      run_full_release:
-        description: 'Run full release process (`no` to skip)'
-        required: true
-        default: 'yes'
-        type: string
+      full_release_extensions:
+        description: |
+          A comma-separated list of extensions for which to run the full release process.
```

Comment:
Can you clarify that this will only create a GitHub tag and attach XCFrameworks?

Reply:
Yes. The tag should be created as part of release process.